packages/jsii-pacmak/lib/targets/go/runtime/class-constructor.ts

```diff
@@ -38,10 +40,11 @@ export class ClassConstructor {
     return `[]${JSII_FQN}{${iFaceList}}`;
   }
 
-  public get argsString(): string {
-    const argsList = this.parent.parameters
-      .map((param) => param.name)
-      .join(', ');
-    return `[]interface{}{${argsList}}`;
+  private get argsString(): string {
+    const argsList = this.parent.parameters.map((param) => param.name);
+    if (argsList.length === 0) {
+      return 'nil /* no parameters */';
```

Comment:
teeny tiny nit:
All the others (no parameters, no overrides etc) emit `nil, // no x`, line comment instead of block, following the `,`.

Reply:
Yeah this is different because the comment is emitted before the comma due to how the code is structured...